model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -117,6 +117,9 @@ data class OrtConfiguration(
     val notifier: NotifierConfiguration = NotifierConfiguration()
 ) {
     companion object : Logging {
+        /** The name of the configuration file with the default values. */
+        private const val DEFAULT_CONFIG_FILE = "/defaults.yml"
```

Comment:
IIRC at some point we decided to not externalize default values to files, but programmatically have them as default arguments. I believe @fviernau's main argument for this was support in test and refactoring.

Reply:
Because of the length of the list, I would not want to define it in `OrtConfiguration`. So, would it be okay to create an object `AllowedProcessEnvironmentVariables` which contains the list and also functionality to deal with regular expressions?

Reply:
> So, would it be okay to create an object `AllowedProcessEnvironmentVariables` which contains the list and also functionality to deal with regular expressions?

Sounds like the best compromise 😒

Reply:
The default values and logic is now contained in an object named `EnvironmentVariableFilter`.